src/field_theory/ratfunc.lean

```diff
@@ -306,6 +310,18 @@ instance [is_domain K] : nontrivial (ratfunc K) :=
 ⟨⟨0, 1, mt (congr_arg to_fraction_ring) $
   by simpa only [← of_fraction_ring_zero, ← of_fraction_ring_one] using zero_ne_one⟩⟩
 
```

Comment:
Commenting here because I can't comment any closer; is the comment on line 285 now wrong?

Reply:
I think that once your recent PR merges then yes, we'll be able to remove those.